src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.less

```diff
@@ -44,6 +44,8 @@
       .list-group-item-text {
         margin-right: 0;
         margin-top: 15px;
+        white-space: unset;
+        font-size: 14px;
```

Comment:
It'd be best to change this to `font-size: 1em;` so that it will scale whenever the base font is changed in the future.